crates/core/c8y_api/src/json_c8y_deserializer.rs

```diff
@@ -405,12 +405,19 @@ pub struct C8yLogfileRequest {
     pub maximum_lines: usize,
 }
 
+/// Type assigned to legacy configuration upload/download
+/// operations (e.g. when the type property is not set in the operation)
+fn legacy_config_type() -> String {
+    "legacy".to_string()
+}
```

Comment:
This is simply the default type. The fact that the legacy c8y API doesn't provide a type should not taint thin-edge naming.

```suggestion
fn default_config_type() -> String {
    "default".to_string()
}
```

Reply:
resolved by 2b1fd5e54014d134befacf2ca7aef7626f7995d0